### PR TITLE
showing articles in chronological order

### PR DESCRIPTION
**Pull request by 0xPranjl**

### CODE REVIEW

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]
This should be done in the `getAllArticlesMetadata` function so that it happens before the page is loaded. It would just make it a little more optimal. 

Thanks for contributing too! Great to have you here ❤️

**Comment by pkspyder007** [comment]
@0xPranjl As Niall said we can move this to server side logic in the `getAllArticlesMetadata` by just sorting on date in frontmatter at the time of returning.

```js
return metadata.sort((a, b) => {
     return (new Date(b.date).valueOf() - new Date(a.date).valueOf());
});
```

**Comment by 0xPranjl** [comment]
thanks @pkspyder007 I have applied this logic

File: pages/articles/index.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import ArticlePreview from "../../components/ArticlePreview/ArticlePreview";
 import Layout from "../../components/Layout/Layout";
 import { getAllArticlesMetadata } from "../../lib/blog";
 import { authors } from "../../config/site_settings";
+import { useEffect, useState } from "react";
+import { Temporal } from "@js-temporal/polyfill";
```
Comment (NiallJoeMaher) [inline pages/articles/index.tsx:8]: Are these imports needed now?

**Review by NiallJoeMaher — APPROVED** [review]

**Comment by NiallJoeMaher** [comment]
@0xPranjl your changes are live! Nice one. https://codu.co/articles

**Comment by 0xPranjl** [comment]
thanks

**Comment by pkspyder007** [comment]
Congrats @0xPranjl on your first pr merge 🥳

**Comment by 0xPranjl** [comment]
thanks @pkspyder007

**Pull request merged**